algorithmic_efficiency/workloads/criteo1tb/criteo1tb_jax/models.py

```diff
@@ -104,5 +103,9 @@ def scaled_init(key, shape, dtype=jnp.float_):
               top_mlp_input)
       if layer_idx < (num_layers_top - 1):
         top_mlp_input = nn.relu(top_mlp_input)
+      if self.dropout_rate > 0.0 and layer_idx == num_layers_top - 2:
```

Comment:
`self.dropout_rate` can be `None`, so in that case this line will lead to an error. I think this is why pytest currently fails, since `dropout_rate` is set to `None` by default [here](https://github.com/mlcommons/algorithmic-efficiency/pull/266/files#diff-724cda3e2dc6d8233317b4fcaa9fb047fe0804003e330ecfec013c8a43d18298R63).

Reply:
good catch, fixed.